api/docs/src/components/SignupForm.jsx

```diff
@@ -12,6 +12,19 @@ import {
 // Taken from https://ui.dev/validate-email-address-javascript/
 const EMAIL_REGEX = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
 
+const HUBSPOT_COOKIE_NAME = "hubspotutk";
+
+// Taken from
+// https://gist.github.com/joduplessis/7b3b4340353760e945f972a69e855d11
+function getCookie(name) {
+  const value = "; " + document.cookie;
+  const parts = value.split("; " + name + "=");
+
+  if (parts.length == 2) {
+    return parts.pop().split(";").shift();
```

Comment:
Did you see the possible improvements at https://gist.github.com/joduplessis/7b3b4340353760e945f972a69e855d11#gistcomment-3668361 ?

Reply:
ah yeah - we're not yet using typescript yet :'( but planning on changing that soon